aiosolr.py

```diff
@@ -76,20 +77,22 @@ def get(self, name, default=None):
 
 
 # TODO Support something other than JSON
-class Client:
+class Client:  # pylint: disable=too-many-instance-attributes
     """Class representing a client connection to Solr."""
 
-    def __init__(  # pylint: disable=dangerous-default-value
+    def __init__(
         self,
         *,
         collection="",
-        connection_url=None,
+        connection_url: str | None = None,
         debug=False,
         host="127.0.0.1",
         port="80",
         scheme="http",
-        timeout=(1, 3),
```

Comment:
I propose a breaking change with the rename of `timeout` to `client_timeout` to disambiguate with `read_timeout` and `write_timeout`, and the type of `client_timeout` is `aiohttp.ClientTimeout` instead of an int or tuple. If this is undesirable, we could also keep `timeout` as-is while adding `read_timeout` and `write_timeout`

Reply:
Proposed change accepted